docs/user_guide.md

```diff
@@ -0,0 +1,66 @@
+.. _user-guide:
+
+👩‍💻 User guide
+==========
+
+📚 **Contents:**
+
+- :ref:`install`
+- :ref:`getting_started`
+- :ref:`catalog-section`
+
+.. _install:
+
+⬇️ Installation
+--------------
+
+Scivision is a Python package. You can install the latest version from [PyPi](https://pypi.org/project/scivision/) with pip:
+
+````bash
+    pip install scivision
+````
+
+.. _getting_started:
+
+🧪 Getting started
+---------------
+
+The creators of scivision envisioned it as enabling users to quickly run several lines of Python code to either find computer vision models that can be run on their scientific image dataset(s), or find datasets that their models can be run on.
+
+In essence, the scivision Python API consists of just **five** lines of code that capture the core functionality of the package. Four of these are documented in the :ref:`api-docs` documentation, with the fifth being the `predict` function from the computer vision models included in the scivision catalog.
```

Comment:
Perhaps "you can do something useful in five lines" rather than "the API has five functions" is more appealing.  The API may grow with additional querying functionality, but a typical use case might only need one or two.

(See comment below too)